packages/eslint-config-base/rules/imports.js

```diff
@@ -142,7 +142,13 @@ module.exports = {
     ...getNoExtraneousDepsOverrides(),
     // This override enables default exports for files that require them.
     {
-      files: ['jest.config.{js,ts,mjs,cjs}', 'vitest.config.{js,ts,mjs,cjs}'],
+      files: [
```

Comment:
Looks good to me. While you're at it, can you add `cjs` to our extensions as well? I was about to do another PR for that.

Reply:
Oh yeah good callout - added